### ISSUE TITLE
Allow zoom gestures on mobile

### ISSUE THREAD

**Issue by ubercj** [e-gineering/movement-challenge-calculator#1]
https://github.com/e-gineering/movement-challenge-calculator/blob/638acc8f6179bd9a9457b26b86f705255f35f961/app.js#L436

Zoom gestures are a critical accessibility feature. Restricting this risks violating [WCAG SC 1.4.4 Resize Text](https://www.w3.org/TR/wcag2mobile-22/#success-criterion-1-4-4-resize-text)
